### PR TITLE
3-create-support-files-and-dashboard-to-answer-bq-23

### LINKED ISSUE #3: Create support files and dashboard to answer BQ 2.3
Acceptance criteria:

-Add Excel file with Mockup Data
-Add Screenshot/PDF of Dashboard developed with Looker
-Add any other relevant documentation

### PR DESCRIPTION
**Pull request by jdcastellanosb73**
3-create-support-files-and-dashboard-to-answer-bq-23.

the task was completed with the .xls, the control board and a link to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Analytics
\ No newline at end of file
+# Analytics
+
+link para el looker studio BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```
Comment (jsurrea) [inline README.md]: Hey can you update this to be in English? Thank you!

**Review by jsurrea — CHANGES_REQUESTED** [review]
Please remember to always use English!

File: README.md (hunk 1 above)
Comment (jsurrea) [reply README.md]: Also with the file names!

Comment (jsurrea) [reply README.md]: Also, to ensure consistency, rename the folder to BQ 2.3

Comment (jsurrea) [reply README.md]: Hey @jdcastellanosb73 ! Can you change the folder name (add the space) and file names (to English) as well?

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Analytics
\ No newline at end of file
+# Analytics
+
+link for the looker studio to answer the BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```
Comment (jsurrea) [inline README.md]: https://lookerstudio.google.com/s/j8pzo3OZxkg is not a public link! Check this:


Uploading Screen Recording 2024-03-22 at 16.03.09.mov…

**Review by jsurrea — CHANGES_REQUESTED** [review]

**Review by SnowArtz — CHANGES_REQUESTED** [review]
I think we should use a different folder structure, i would use meanwhile the next Folders:

excel_mockups
dashboards

And the name conventions of the excel: bqx.y.xlsx
And the name convention of the dashboard: bqx.y.pdf

**Review by SnowArtz — CHANGES_REQUESTED** [review]
Please Upload a Python file instead a Jupyter Notebook!

**Review by SnowArtz — CHANGES_REQUESTED** [review]
Please Add a small text into the upper left related to your BQ number: 2,3

**Review by SnowArtz — CHANGES_REQUESTED** [review]
You havent delete the notebook.

File: README.md [hunk 3]
```diff
@@ -1 +1,3 @@
-# Analytics
\ No newline at end of file
+# Analytics
+
+link for the looker studio to answer the BQ 2.3: https://lookerstudio.google.com/reporting/9281694c-2631-46f5-8aab-0b23cb568ee9
```
Comment (SnowArtz) [inline README.md]: This link now is with editor permissions!! Take care,

**Review by SnowArtz — CHANGES_REQUESTED** [review]

**Review by Laudarias — APPROVED** [review]
Link is working correctly now

**Review by SnowArtz — APPROVED** [review]
Nice!

**Review by iGotty — APPROVED** [review]

**Review by jsurrea — APPROVED** [review]

**Pull request merged**